File: RenetikObjc/Library/Classes/Core/Classes/View/CSTextField.m
//
// Created by Rene Dohan on 20/11/13.
// Copyright (c) 2013 creative_studio. All rights reserved.
//

#import "CSTextField.h"

@implementation CSTextField {

}

- (CGRect)caretRectForPosition:(UITextPosition *)position {
    return self.hideCursor ? CGRectZero : [super caretRectForPosition:position];
}

@end

File: Renetik/Renetik/Classes/Core/Classes/View/CSTextField.swift
//
// Created by Rene Dohan on 12/22/19.
//

import UIKit

public class CSTextField: UITextField {
    public var hideCursor = false

    override public func caretRect(for position: UITextPosition) -> CGRect {
        hideCursor ? .zero : super.caretRect(for: position)
    }

}
